src/components/DoiMetadata/DoiMetadata.test.tsx

```diff
@@ -65,6 +67,13 @@ describe('DoiMetadata Component', () => {
       .should('be.visible')
   })
 
+  it('tags', () => {
```

Comment:
Maybe we just need a test for filed of science here as the content come from other source.

Reply:
Makes sense.